x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/create_threat_signals.ts

```diff
@@ -90,12 +91,45 @@ export const createThreatSignals = async ({
     perPage,
   });
 
+  const getMatchedThreats: GetMatchedThreats = async (ids) => {
+    const matchedThreatsFilter = {
+      query: {
+        bool: {
+          filter: {
+            ids: { values: ids },
+          },
+        },
+      },
+    };
+    const threatResponse = await getThreatList({
+      callCluster: services.callCluster,
+      exceptionItems: exceptionItems ?? [],
+      threatFilters: [...(threatFilters ?? []), matchedThreatsFilter],
```

Comment:
I don't think either exceptionItems or threatFilters can be `null/undefined` unless I'm missing something. You should be able to clean up this already pretty clean code by removing the `??`:

```ts
      exceptionItems,
      threatFilters: [...threatFilters, matchedThreatsFilter],
```

Reply:
Good call, thank you!